runner.sh

```diff
@@ -1,19 +1,13 @@
 #!/bin/bash
 
 export TASKS="$(node ./runner/check-aws-ecs-tasks.js)"
+echo $TASKS
 
-if [[ $TASKS == 0 ]]; then
+if [[ $TASKS == "No running tasks found" ]]; then
   cd $CAS_PATH && npm run start
-  exit_code=$?
-  if [[ $exit_code != 0 ]]; then
+  EC=$?
```

Comment:
why the change from `exit_code` to `EC`?  I tend to prefer more meaningful names, even if they are a bit more verbose